pkg/specgen/generate/kube/volume.go

```diff
@@ -45,7 +45,7 @@ type KubeVolume struct {
 	// This is only used when there are volumes in the yaml that refer to a configmap
 	// Example: if configmap has data "SPECIAL_LEVEL: very" then the file name is "SPECIAL_LEVEL" and the
 	// data in that file is "very".
-	Items map[string]string
+	Items map[string][]byte
```

Comment:
@Luap99 can you double check whether this is a breaking change?

I don't think so, but I have missed some in the past, so I prefer a head nod from you.

Reply:
If anyone is using this package as a library and accesses this field, this is a breaking change. I highly doubt that anyone is actually doing this, and [I can't find any occurrences in public repositories](https://sourcegraph.com/search?q=context:global+%22github.com/containers/podman/v4/pkg/specgen/generate/kube%22&patternType=standard).